src/python/bot/tasks/fuzz_task.py

```diff
@@ -1673,8 +1677,8 @@ def run(self):
 
     # Check if we have an application path. If not, our build failed
     # to setup correctly.
-    app_path = environment.get_value('APP_PATH')
-    if not app_path:
+    if (environment.get_value('APP_NAME') and
```

Comment:
Why are we depending on environment.get_value('APP_NAME') here ? What is the case where we dont have APP_NAME, but dont have APP_PATH ?

Reply:
This is for the case of the fuzzing pipeline, where we don't have APP_NAME = launcher.py. otherwise we bail because APP_PATH isn't set.